cmd/main.go

```diff
@@ -57,7 +73,14 @@ func main() {
 	})
 
 	e.GET("/players", func(c echo.Context) error {
-		return c.Render(http.StatusOK, "players", nil)
+		var players []*database.Player
+		result := db.Find(&players) // SELECT * FROM Players;
```

Comment:
can you move this out to a another layer?

Reply:
Sure, done.